### ISSUE TITLE
Rebuild sensors.

### ISSUE THREAD

**Issue by DaloLorn** [Alarcarr/ABEM-for-Heralds#5]
As stated above.

**Comment by DaloLorn** [comment]
Uhh, we already have all the required code - it's just a matter of restoring the sensor-related subsystems, techs and buildings, and balancing them as necessary.

**Comment by Alarcarr** [comment]
I think its better we leave this for post release .SInce it gonna give u huge headache , and lots of designs rework.Susbystems etc.

**Comment by DaloLorn** [comment]
No, I mean this is best assigned to you because you're the balance guy. :P

I can alter your designs to add sensors, and restore the content, but the balancing is probably better left in your hands.

**Comment by Alarcarr** [comment]
Ah ok in anycase we leave this for next version.

**Comment by Alarcarr** [comment]
First thing we need to discuss is what role sensor should fill , and what effect we want them to have in battles or other interactions.
